results.yml

```
@@ -879,59 +879,58 @@ translations:
 
   - key: hints.hint_hint
     t: >
-      You'll find little hints like this one throughout the survey results
-      that will point out extra features and details.
+      Você encontrará pequenas dicas como esta ao longo dos resultados da pesquisa
+      que apontará recursos e detalhes extras.
 
   - key: hints.units_switcher_hint
     t: >
-      You can use the segmented control at the bottom of each block to switch
-      between different units to get an alternate view of the same data.
+      Você pode usar o controle segmentado na parte inferior de cada bloco para alternar
+      entre diferentes unidades para obter uma visão alternativa dos mesmos dados.
 
   - key: hints.export_hint
     t: >
-      The <span class="hint-icon hint-export-button">Data</span> tab lets you view the raw
-      data for any chart, view it as JSON, or get a GraphQL query you can
-      run against our public API.
+      A guia <span class="hint-icon hint-export-button">Dados</span permite visualizar os dados brutos
+      dados para qualquer gráfico, visualize-o como JSON ou obtenha uma consulta GraphQL que você pode
```

Palavra repetida aqui

```suggestion
      para qualquer gráfico, visualize-o como JSON ou obtenha uma consulta GraphQL que você pode
```